### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by lhvk**

### CODE REVIEW

File: cypress/e2e/1-getting-started/todo.cy.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
+// the power of writing tests in Cypress.
+//
+// To learn more about how Cypress works and
+// what makes it such an awesome testing tool,
+// please read our getting started guide:
+// https://on.cypress.io/introduction-to-cypress
+
+describe("example to-do app", () => {
```
Comment (Fermain) [inline cypress/e2e/1-getting-started/todo.cy.js:14]: You have these in `gitignore` but they are still being tracked. Try deleting them from your project, committing that delete then adding them again

File: cypress/e2e/3-testing/login-logout.cy.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const user = {
+  email: "lhvk@stud.noroff.no",
```
Comment (Fermain) [inline cypress/e2e/3-testing/login-logout.cy.js:2]: A good thing to put in .env

**Review by Fermain — CHANGES_REQUESTED** [review]
Really nice work, if you can remove the remaining example specs this should be ready to go

File: cypress/e2e/1-getting-started/todo.cy.js (hunk 1 above)
Comment (lhvk) [reply cypress/e2e/1-getting-started/todo.cy.js:14]: Done

File: cypress/e2e/3-testing/login-logout.cy.js (hunk 2 above)
Comment (lhvk) [reply cypress/e2e/3-testing/login-logout.cy.js]: After trying several methods that did not work, I settled with a cypress.env.json file where I stored my sensitive data. This file is untracked, but I added an example file for others to use. At least the email and password is now dynamic for whoever want to test it out. I hope this is sufficient. Thanks

**Pull request closed without merging**